etc/profile-m-z/youtube-viewers-common.profile

```diff
@@ -0,0 +1,56 @@
+# Firejail profile for youtube-viewer clones
+# Description: common profile for Trizen's Youtube viewers
+# This file is overwritten after every install/update
+quiet
```

Comment:
`quiet` in a `*-common.profile` makes no sense as `quiet` *must* be the first command.